src/contracts/Event/NameSchema/AbstractSchemaEvent.php

```diff
@@ -6,48 +6,35 @@
  */
 declare(strict_types=1);
 
-namespace Ibexa\Contracts\Core\Event;
+namespace Ibexa\Contracts\Core\Event\NameSchema;
 
-use Ibexa\Contracts\Core\Repository\Values\Content\Content;
 use Symfony\Contracts\EventDispatcher\Event;
 
-final class ResolveUrlAliasSchemaEvent extends Event
+abstract class AbstractSchemaEvent extends Event
 {
     /** @var array<string, array> */
-    private array $schemaIdentifiers;
+    protected array $schemaIdentifiers;
```

Comment:
Implementation classes should only interact with these using methods.
```suggestion
    private array $schemaIdentifiers;
```